godump.go

```diff
@@ -278,14 +294,19 @@ func printDumpHeader(out io.Writer, skip int) {
 var callerFn = runtime.Caller
 
 // findFirstNonInternalFrame iterates through the call stack to find the first non-internal frame.
-func findFirstNonInternalFrame() (string, int) {
-	for i := 2; i < 10; i++ {
+func findFirstNonInternalFrame(skip int) (string, int) {
+	for i := initialCallerSkip; i < defaultMaxStackDepth; i++ {
 		pc, file, line, ok := callerFn(i)
 		if !ok {
 			break
 		}
 		fn := runtime.FuncForPC(pc)
-		if fn == nil || !strings.Contains(fn.Name(), "godump") {
+		if fn == nil || !strings.Contains(fn.Name(), "github.com/goforj/godump") || !strings.Contains(file, "godump.go") {
```

Comment:
This should be a constant as the repository may be forked.

Another way to consider it is to only look for fn.Name and stop when facing "godump.Test"